modules/ossm-security-mtls.adoc

```diff
@@ -15,17 +15,17 @@ By default, {ProductName} is set to permissive mode, where the sidecars in {Prod
 [id="ossm-security-enabling-strict-mtls_{context}"]
 == Enabling strict mTLS across the mesh
 
-If your workloads do not communicate with services outside your mesh and communication will not be interrupted by only accepting encrypted connections, you can enable mTLS across your mesh quickly. Set `spec.istio.global.mtls.enabled` to `true` in your `ServiceMeshControlPlane` resource. The operator creates the required resources.
+You can quickly enable mTLS across your mesh if your workloads do not communicate with outside services, and communication will not be interrupted by accepting only encrypted connections. Set `spec.security.mutualTLS.controlPlane.enable` to `true` in your `ServiceMeshControlPlane` resource. The operator creates the required resources.
```

Comment:
Shouldn't this section mention both settings for mTLS (`spec.security.controlPlane.mtls` and `spec.security.dataPlane.mtls`)?

Also not that the field name is no longer `mutualTLS` and that there is no `enable` field. In both `controlPlane` and `dataPlane`, the field is now just `mtls`.

Reply:
I think that content has been addressed in https://github.com/openshift/openshift-docs/pull/26354 https://github.com/openshift/openshift-docs/pull/26354/files#r516721277 I think I'd like to do the work in that PR and rebase this PR after that one's merged.